### PR TITLE
LPD-34657 Remove FF LPD-29637 of Allow users to activate/deactivate a System Validation

### PR DESCRIPTION
**Pull request by icgsbr**
https://liferay.atlassian.net/browse/LPD-34657

### CODE REVIEW

**Comment by icgsbr** [comment]
ci:test:sf

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java [hunk 1]
```diff
@@ -756,11 +755,9 @@ else if (Objects.equals(
...
 				NAME_COMPOSITE_KEY_OBJECT_FIELD_ID);
 
-		if (FeatureFlagManagerUtil.isEnabled("LPD-29637")) {
-			allowedObjectValidationRuleSettingNames.add(
-				ObjectValidationRuleSettingConstants.
-					NAME_ALLOW_ACTIVE_STATUS_UPDATE);
-		}
+		allowedObjectValidationRuleSettingNames.add(
+			ObjectValidationRuleSettingConstants.
+				NAME_ALLOW_ACTIVE_STATUS_UPDATE);
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java:760]: Can you add the constant to the allowedObjectValidationRuleSettingNames? sort it, please.

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by icgsbr** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:object

**Comment by marcelabc** [comment]
ci:test:playwright

**Comment by icgsbr** [comment]
ci:test:sf

**Comment by icgsbr** [comment]
ci:test:relevant

**Comment by icgsbr** [comment]
ci:reopen

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:report:6910724

**Comment by marcelabc** [comment]
ci:report:6841700

**Comment by marcelabc** [comment]
ci:forward:force

**Comment by marcelabc** [comment]
I checked the results, and there is no related failure.
@icgsbr, can you forward this manually to Brian? I think there is a bug in the label of the stable results.
Please update this PR and the ticket with the new PR.

**Comment by marcelabc** [comment]
ci:forward:force

**Pull request closed without merging**